web/static/web/css/header.css

```diff
@@ -1,144 +1,332 @@
+    grid-template-columns: 1fr auto 1fr;
     width: 100%;
-    height: 120px;
-    background-color: rgb(33, 43, 52);
+    height: var(--header-height);
+    background-color: rgb(34, 43, 52);
 }
 
-.nav .menu {
-    align-self: center;
-    margin: 0;
-    padding: 0 30px;
+#header .nav-item .nav-item-text,
+#nav-user-dropdown .dropdown-item .dropdown-item-text,
+.lang-link .item .text {
+    color: #DCDDDE;
 }
 
-.nav .logo {
+#header .nav-item:hover .nav-item-text,
+#nav-user-dropdown .dropdown-item:hover .dropdown-item-text,
+.lang-link .item:hover .text {
+    color: #FFF;
+}
+
+#header div.logo-centering {
+    display: none;
+}
+
+#header .logo {
+    display: grid;
+    grid-template-columns: var(--logo-padding) var(--logo-space) minmax(var(--nav-space-between), var(--logo-padding));
     align-self: center;
-    padding: 0 30px;
 }
 
-.nav .logo img {
+#header .logo img {
+    margin: auto;
+    max-height: 50px;
     object-fit: contain;
 }
 
-.limg {
+#header .logo .limg.large {
+    display: block;
+}
+
+#header .logo .limg.medium {
     display: none;
 }
 
-.limg.large {
-    display: inline;
+#header .logo .limg.small {
+    display: none;
 }
 
-.burger {
+#header #burger {
     display: none;
+    align-self: center;
+    justify-self: end;
+    margin-right: var(--burger-margin);
     margin-bottom: -10px;
 }
 
-.burger:hover {
+#header #burger:hover {
     cursor: pointer;
 }
 
-.burger {
-    align-self: center;
+#header #burger i {
+    color: #FFFFFF;
+    font-size: 30px;
+}
+
+#header #nav {
     justify-self: center;
+    font-size: var(--font-size);
 }
 
-.burger i {
-    color: #FFFFFF;
-    font-size: 30px;
+#header #nav .nav-item,
+#header #side-nav .nav-item {
+    height: calc(var(--font-size) + 2em);
+    margin: calc(var(--nav-space-between) / 2);
+    padding: 0;
+    border-radius: 0;
+    justify-content: center;
 }
 
-.nav .ui.inverted.menu .item {
-    color: rgba(255, 255, 255, .7) !important;
+#header #nav .nav-item:hover,
+#header #side-nav .nav-item:hover,
+#nav-user-dropdown.active {
+    /* Disables property from Semantic UI */
+    background-color: transparent;
 }
 
-.lang-form {
-    display: none;
+#header #nav .nav-item .bubble-background,
+#header #side-nav .nav-item .bubble-background {
+    position: absolute;
+    width: 100%;
+    height: var(--font-size);
+    border-radius: 2em;
+    z-index: 1;
+    transition: width var(--dur) linear, height var(--dur) linear, background-color var(--dur) ease-out;
+}
+
+#header #nav .nav-item:hover .bubble-background,
+#header #side-nav .nav-item:hover .bubble-background {
+    width: calc(100% + var(--bubble-background-extra-width));
+    height: calc(var(--font-size) + 2em);
+    z-index: 2;
+}
+
+#header #nav .nav-item .nav-item-text,
+#header #side-nav .nav-item .nav-item-text {
+    z-index: 3;
+    transition: color var(--dur) ease;
+}
+
+#header #nav .nav-item:nth-child(1):hover .bubble-background {
+    background-color: rgba(239, 184, 14, 0.9);
```

Comment:
Nice suggestion! Don't know why I didn't think of that :p Will change to using classes for the colors instead. Unsure what you mean by "set the color just on hover", though..

Reply:
Now, you just set the color on hover, since `.bubble-background` has a width of 100% when not hovered (for the animation). Meaning that if you just make a class, e.g:
```css
.make_bg_yellow_80 {
    background-color: rgba(239, 184, 14, 0.8);
}
```

and doing `... class="make_bg_yellow_80 bubble-background"...` would not work correctly. And it is nice to have generic classes for the colors in the profile manual (graphical manual)(Eventually all of the colors in it should be included in the index.css file, with respective variables as well). 

If you make `.bubble-background` have `visibility: hidden` when not hovered, and adding the transition `visibility var(--dur) ease-out` you can have the background color set even when not hovered.